src/components/Calender.vue

```diff
@@ -1,38 +1,49 @@
 <template>
-	<section v-if="listToDisplay != undefined && listToDisplay.size > 0" id="calender">
-		<component 
+	<section
+		v-if="listToDisplay != undefined && listToDisplay.size > 0"
+		id="calender"
+	>
+		<component
 			v-for="calVal in calender"
-			:is="calVal.isPadding ? 'div' : 'a'" 
+			:is="calVal.isPadding ? 'div' : 'a'"
 			:key="calVal.date"
 			:class="{
-				'today': isToday(calVal.date),
-				'padding': calVal.isPadding,
-				'date': true,
+				today: isToday(calVal.date),
+				padding: calVal.isPadding,
+				date: true,
 			}"
 			:title="(filmExist(calVal.date)) ? listToDisplay.get(calVal.date)!.name : ''"
 			:href="(filmExist(calVal.date)) ? `https://letterboxd.com${listToDisplay.get(calVal.date)!.slug}` : ''"
 			target="_blank"
 		>
 			<template v-if="!calVal.isPadding">
 				<h3>
-					{{ calVal.date + 1 }}
+					<div class="number">{{ calVal.date + 1 }}</div>
+					<div class="title">
+						{{ (filmExist(calVal.date)) ? listToDisplay.get(calVal.date)!.name : '' }}
+					</div>
 				</h3>
+
 				<p
 					v-for="[username, filmList] in userFilmList.entries()"
-					:class="filmList.get(listToDisplay.get(calVal.date)?.name ?? '')?.toLowerCase() ?? ''"
+					:class="
+						filmList
+							.get(listToDisplay.get(calVal.date)?.name ?? '')
+							?.toLowerCase() ?? ''
+					"
```

Comment:
I'm not finding this change an improvement to readability

Reply:
Ya I don't know how to make my styler chill